### DISCUSSION TITLE
Use Python NLTK on Vercel in Next.js App?

### DISCUSSION THREAD

**Discussion by lancejpollard** [vercel/community#3575] [Help]
### Summary

Can I have this sort of script in a handler in a Next.js app with Python API?

```py
import nltk

nltk.download('punkt')
nltk.download('averaged_perceptron_tagger')

sentence = "I care about caring"
tokens = nltk.word_tokenize(sentence)
pos_tags = nltk.pos_tag(tokens)

for word, pos in pos_tags:
    print(f'{word}: {pos}')
```

Specifically I'm worried about the `nltk.download('punkt')` and other download calls, since Vercel is serverless. Will this work? I would like to avoid building something locally to find out it won't work on Vercel deployment. I am looking at these two sources for examples:

- https://vercel.com/guides/how-to-use-python-and-javascript-in-the-same-application
- https://github.com/vercel/examples/tree/main/python/flask

If it works, can I just place a `requirements.txt` at the base of the project, along with the instructions in the first link?

### Steps to Reproduce

- https://vercel.com/guides/how-to-use-python-and-javascript-in-the-same-application
- https://github.com/vercel/examples/tree/main/python/flask

**Comment by GH0STH4CKER** [comment]
Uploaded folder nltk_data/tokenizers/punkt to github and added path of nltk data.


`nltk.data.path.append("/var/task/api/nltk_data/")`
